cmd/kubectl_token.go

```diff
@@ -434,45 +442,45 @@ func samlAuth(input *LoginInput, client *http.Client) (managementClient.Token, e
 
 	tokenURL := fmt.Sprintf(authTokenURL, input.server, id)
 
-	req, err := http.NewRequest(http.MethodGet, tokenURL, bytes.NewBuffer(nil))
+	getReq, err := http.NewRequest(http.MethodGet, tokenURL, bytes.NewBuffer(nil))
 	if err != nil {
 		return token, err
 	}
+	getReq.Header.Set("content-type", "application/json")
+	getReq.Header.Set("accept", "application/json")
 
-	req.Header.Set("content-type", "application/json")
-	req.Header.Set("accept", "application/json")
-
-	client.Timeout = 300 * time.Second
```

Comment:
This seems unnecessary. There is nothing specific or unusual about requesting an auth token that might require a custom 5 min timeout.